src/server/StudioManagementSystem.Core/Dtos/GroupDto.cs

```diff
@@ -6,6 +6,6 @@ public record GroupDto(
     string Description,
     MemberInfoDto MemberInfo,
     List<PreferenceDto> Preferences,
-    ProjectDto? Project,
+    string? Project,
```

Comment:
this means the frontend won't receive project details, including the project ID (which would be needed for page navigation). Why change this?